Update docs and validation to account for dynamic implementation (#129)
AlaskaAirlines/auro-radio: demo/api.md

|---------|------------|------------------------------------------------|
| [reset](#reset) | `(): void` | Method for a total reset of the radio element. |

## Events

| Event                       | Description                                      |
|-----------------------------|--------------------------------------------------|

> | `auroFormElement-validated` | Notifies that the `validity` and `errorMessage` values have changed. |

Reviewer: This description is not necessarily true. It can validate and still show the same message.

```suggestion
| `auroFormElement-validated` | Notifies that the element has been validated.
```

> | `auroRadioGroup-helpText`   | Notifies that the `setCustomValidity` value has changed. |

Reviewer: Didn't this get removed?

> | `auroRadioGroup-validated`  | Notifies that the `validity` value has changed.  |

Reviewer: Isn't this gone now in favor of the generic message?
